java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/utils/PropertiesSample.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.arrow.driver.jdbc.utils;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.Properties;
+
+import javax.annotation.Nullable;
+
+import org.apache.arrow.driver.jdbc.FlightServerTestRule;
+import org.apache.arrow.util.Preconditions;
+
+
+/**
+ * {@link Properties} wrapper used for testing. Uses sample values.
+ *
+ * @see FlightServerTestRule
+ * @deprecated not updatable to match dinamic server allocation.
+ */
+@Deprecated
```

Comment:
Ditto here, why add a class only to immediately deprecate it? (I guess things are in the middle of a refactoring here?)

Reply:
Yes! This is still work in progress and we will have to clean up